armi/reactor/converters/uniformMesh.py

```diff
@@ -185,11 +185,9 @@ def _decuspAxialMesh(self):
         # filter fuel material boundaries to mininum mesh size
         filteredBottomFuel, filteredTopFuel = self._getFilteredFuelTopAndBottom()
         (
-            filteredBottomControl,
-            filteredTopControl,
+            materialBottoms,
+            materialTops,
         ) = self._getFilteredControlTopAndBottom(filteredBottomFuel, filteredTopFuel)
-        materialBottoms = filteredBottomFuel + filteredBottomControl
-        materialTops = filteredTopFuel + filteredTopControl
 
         # combine the bottoms and tops into one list with bottom preference
         allMatBounds = materialBottoms + materialTops
```

Comment:
Am I correct in understanding that this list now only includes bottoms and tops from the control assemblies? I can't quite tell. 

If that understanding is not correct, then I don't see why it used to be necessary to combine the lists, but now is not.

Reply:
It wasn't necessary before. Maybe we should change the name of `getFilteredControlTopAndBottom`. It returns lists that include whatever was already in `filteredBottomFuel` and `filteredTopFuel`, respectively.

Reply:
We had an offline conversation about consolidating and simplifying into a single `_getFilteredMeshTopAndBottom` method.